README.md
mx-codecs
=========

tool for installing restricted deb files.


debian/changelog
mx-codecs (23.7.01) mx; urgency=medium

  * translation update

 -- Dolphin Oracle <dolphinoracle@gmail.com>  Thu, 20 Jul 2023 15:07:45 -0400

mx-codecs (23.7) mx; urgency=medium

  * Refresh translations
  * Update policy file

 -- Adrian <adrian@mxlinux.org>  Sat, 01 Jul 2023 19:51:42 -0400

mx-codecs (23.5) mx; urgency=medium

  * Update logo
  * Refresh translations

 -- Adrian <adrian@mxlinux.org>  Wed, 17 May 2023 22:16:13 -0400

mx-codecs (23.03.02) mx; urgency=medium

  * Detect proxy
  * Update translations

 -- Adrian <adrian@mxlinux.org>  Sat, 11 Mar 2023 15:10:51 -0500

mx-codecs (23.03.01) mx; urgency=medium

  * make work with bookworm 

 -- Dolphin Oracle <dolphinoracle@gmail.com>  Sat, 04 Mar 2023 22:04:29 -0500

mx-codecs (22.10.01) mx; urgency=medium

  * update translations

 -- Dolphin Oracle <dolphinoracle@gmail.com>  Wed, 26 Oct 2022 19:57:53 -0400

mx-codecs (22.8.01) mx; urgency=medium

  * Fix: launching documents from Help (fixes a problem in Plasma)

 -- Adrian <adrian@mxlinux.org>  Mon, 22 Aug 2022 19:09:58 -0400

mx-codecs (22.8) mx; urgency=medium

  * Fix: launching Firefox from Help 

 -- Adrian <adrian@mxlinux.org>  Wed, 17 Aug 2022 21:47:07 -0400

mx-codecs (22.7) mx; urgency=medium

  * translation refresh 

 -- Dolphin Oracle <dolphinoracle@gmail.com>  Sat, 23 Jul 2022 11:22:35 -0400

mx-codecs (22.6) mx; urgency=medium

  * Preserve user theme when launching program as root
  * Make sure HOME and XDG_RUNTIME_DIR are correct when launching as root

 -- Adrian <adrian@mxlinux.org>  Tue, 14 Jun 2022 18:16:17 -0400

mx-codecs (21.7) mx; urgency=medium

  * Download only header not the entire page for network check
  * Use shorter timeout for network check than the default 

 -- Adrian <adrian@mxlinux.org>  Thu, 15 Jul 2021 14:15:36 -0400

mx-codecs (21.1.01) mx; urgency=medium

  * update to only install non-free codecs on x86 architectures 

 -- Dolphin Oracle <dolphinoracle@gmail.com>  Tue, 26 Jan 2021 23:14:52 -0500

mx-codecs (21.1) mx; urgency=medium

  * remove wget dependency
  * minor fixes

 -- Adrian <adrian@mxlinux.org>  Mon, 04 Jan 2021 03:31:29 -0500

mx-codecs (19.9.01) mx; urgency=medium

  * update translations

 -- Adrian <adrian@mxlinux.org>  Sun, 29 Sep 2019 12:57:37 -0400

mx-codecs (19.9) mx; urgency=medium

  * update translations
  * remove libcmd dependency

 -- Adrian <adrian@mxlinux.org>  Sun, 01 Sep 2019 02:10:54 -0400

mx-codecs (19.8) mx; urgency=medium

  * added offline help file

 -- Adrian <adrian@mxlinux.org>  Wed, 21 Aug 2019 13:42:47 -0400

mx-codecs (19.7.1) mx; urgency=medium

  * update icon

 -- Adrian <adrian@mxlinux.org>  Mon, 08 Jul 2019 12:05:46 -0400

mx-codecs (19.7) mx; urgency=medium

  * fix window buttons for Buster

 -- Adrian <adrian@mxlinux.org>  Sun, 07 Jul 2019 13:35:00 -0400

mx-codecs (19.6.01) mx; urgency=medium

  * added buster support 

 -- Dolphin Oracle <dolphinoracle@gmail.com>  Sun, 30 Jun 2019 15:13:30 -0400

mx-codecs (19.4.3) mx; urgency=medium

  * revised install of texture packs (remove preinstalled ones first) 

 -- Dolphin Oracle <dolphinoracle@gmail.com>  Sat, 27 Apr 2019 12:48:41 -0400

mx-codecs (19.4.2) mx; urgency=medium

  * additional fix for Jessie

 -- Adrian <adrian@mxlinux.org>  Sat, 20 Apr 2019 10:55:02 -0400

mx-codecs (19.4.1) mx; urgency=medium

  * fix for older releases

 -- Adrian <adrian@mxlinux.org>  Fri, 19 Apr 2019 15:13:43 -0400

mx-codecs (19.4) mx; urgency=medium

  * updated translations

 -- Adrian <adrian@mxlinux.org>  Fri, 05 Apr 2019 11:28:10 -0400

mx-codecs (18.11.1) mx; urgency=medium

  * updated translations

 -- Adrian <adrian@mxlinux.org>  Fri, 30 Nov 2018 12:01:03 -0500

mx-codecs (18.11) mx; urgency=medium

  * add libtxc-dxtn0 from deb-multi to install routine 

 -- dolphin <dolphin@littlebyte>  Sat, 17 Nov 2018 10:04:48 -0500

mx-codecs (18.10) mx; urgency=medium

  * updated translations

 -- Adrian <adrian@mxlinux.org>  Sun, 14 Oct 2018 18:35:20 -0400

mx-codecs (18.8) mx; urgency=medium

  * rm gksu dependency
  * fix icon other themes

 -- Adrian <adrian@mxlinux.org>  Sat, 11 Aug 2018 13:34:37 -0400

mx-codecs (18.7) mx; urgency=medium

  * don't run if dpkg is locked
  * lock dpkg for other programs

 -- Adrian <adrian@mxlinux.org>  Fri, 13 Jul 2018 12:00:11 -0400

mx-codecs (18.6.1) mx; urgency=medium

  * added changelog button

 -- Adrian <adrian@mxlinux.org>  Sun, 24 Jun 2018 10:41:15 -0400

mx-codecs (18.6) mx; urgency=medium

  * new translations

 -- Adrian <adrian@mxlinux.org>  Mon, 18 Jun 2018 12:48:23 -0400

mx-codecs (18.3) mx; urgency=medium

  * new translations

 -- Adrian <adrian@mxlinux.org>  Fri, 02 Mar 2018 10:29:07 -0500

mx-codecs (18.2) mx; urgency=medium

  * new translations

 -- Adrian <adrian@mxlinux.org>  Sat, 10 Feb 2018 01:39:12 -0500

mx-codecs (17.12) mx; urgency=medium

  * new translations

 -- Adrian <adrian@mxlinux.org>  Mon, 04 Dec 2017 14:30:12 -0500

mx-codecs (17.11+1) mx; urgency=medium

  * fix LANG detection

 -- Adrian <adrian@mxlinux.org>  Sat, 11 Nov 2017 18:36:23 -0500

mx-codecs (17.11) mx; urgency=medium

  * add FR help

 -- Adrian <adrian@mxlinux.org>  Fri, 10 Nov 2017 10:59:35 -0500

mx-codecs (17.7mx16) mx; urgency=medium

  * new icons

 -- Dolphin <dolphinoracle@gmain.com>  Sun, 15 Oct 2017 09:38:58 -0400

mx-codecs (17.6mx16) mx; urgency=medium

  * updated translations

 -- Adrian <adrian@mxlinux.org>  Fri, 02 Jun 2017 13:14:58 -0400

mx-codecs (17.5mx16) mx; urgency=medium

  * update icons

 -- Adrian <adrian@mxlinux.org>  Wed, 10 May 2017 11:44:37 -0400

mx-codecs (17.3mx16) mx; urgency=medium

  * new translations

 -- Adrian <adrian@mxlinux.org>  Thu, 30 Mar 2017 10:01:02 -0400

mx-codecs (16.11mx16+1) mx; urgency=medium

  * new translations

 -- Adrian <adrian@mxlinux.org>  Wed, 30 Nov 2016 10:37:15 -0500

mx-codecs (16.11mx16) mx; urgency=medium

  * new translations

 -- Adrian <adrian@mxlinux.org>  Wed, 16 Nov 2016 11:43:23 -0500

mx-codecs (16.10mx16) mx; urgency=medium


translations/mx-codecs_am.ts
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="am_ET">
<context>
    <name>MainWindow</name>
    <message>
        <location filename="../mainwindow.ui" line="14"/>
        <source>MX Codecs Installer</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.ui" line="27"/>
        <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;This application allows you to install restricted codecs that permit advanced video and audio functions.&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;In some jurisdictions, their distribution may be limited. Please check the regulations in your area. &lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Do you assume legal responsibility for downloading these codecs?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.ui" line="44"/>
        <source>Downloading codecs files</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.ui" line="100"/>
        <source>Cancel any changes then quit</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.ui" line="103"/>
        <source>Cancel</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.ui" line="110"/>
        <source>Alt+N</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.ui" line="129"/>
        <source>Begin codec download and installation</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.ui" line="132"/>
        <source>OK</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.ui" line="193"/>
        <source>About this application</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.ui" line="196"/>
        <source>About...</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.ui" line="203"/>
        <source>Alt+B</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.ui" line="232"/>
        <source>Display help </source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.ui" line="235"/>
        <source>Help</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.ui" line="242"/>
        <source>Alt+H</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.cpp" line="105"/>
        <location filename="../mainwindow.cpp" line="146"/>
        <location filename="../mainwindow.cpp" line="166"/>
        <location filename="../mainwindow.cpp" line="209"/>
        <location filename="../mainwindow.cpp" line="218"/>
        <location filename="../mainwindow.cpp" line="241"/>
        <location filename="../mainwindow.cpp" line="282"/>
        <source>Error</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.cpp" line="106"/>
        <source>Internet is not available, won&apos;t be able to download the list of packages</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.cpp" line="121"/>
        <source>Error downloading %1</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.cpp" line="147"/>
        <source>There was an error writing file: %1. Please check if you have enough free space on your drive</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.cpp" line="166"/>
        <source>Could not create temp directory. </source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.cpp" line="179"/>
        <location filename="../mainwindow.cpp" line="186"/>
        <location filename="../mainwindow.cpp" line="195"/>
        <location filename="../mainwindow.cpp" line="221"/>
        <source>&lt;b&gt;Running command...&lt;/b&gt;&lt;p&gt;</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.cpp" line="179"/>
        <source>downloading Packages.gz from &apos;main&apos;</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.cpp" line="186"/>
        <source>downloading Packages.gz from &apos;non-free&apos;</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.cpp" line="195"/>
        <source>downloading Packages.gz from i386 &apos;main&apos;</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.cpp" line="201"/>
        <source>&lt;b&gt;Download Finished.&lt;/b&gt;</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.cpp" line="209"/>
        <source>Cannot connect to the download site</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.cpp" line="218"/>
        <source>Cannot connect find %1 package</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.cpp" line="238"/>
        <source>Installing downloaded files</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.cpp" line="241"/>
        <source>No downloaded *.debs files found.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.cpp" line="259"/>
        <source>&lt;b&gt;Installing...&lt;/b&gt;&lt;p&gt;</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.cpp" line="261"/>
        <source>Error installing %1</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.cpp" line="266"/>
        <source>Fix missing dependencies...</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.cpp" line="269"/>
        <source>Error running %1 command</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.cpp" line="274"/>
        <source>&lt;b&gt;Installation process has finished&lt;/b&gt;</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.cpp" line="278"/>
        <source>Finished</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.cpp" line="279"/>
        <source>Codecs files have been downloaded and installed successfully.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.cpp" line="282"/>
        <source>Process finished. Errors have occurred during the installation.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.cpp" line="291"/>
        <source>About MX Codecs</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.cpp" line="292"/>
        <source>Version: </source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.cpp" line="294"/>
        <source>Simple codecs downloader for MX Linux</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.cpp" line="296"/>
        <source>Copyright (c) MX Linux</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.cpp" line="297"/>
        <source>%1 License</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mainwindow.cpp" line="308"/>
        <source>%1 Help</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>QObject</name>
    <message>
        <location filename="../about.cpp" line="52"/>
        <source>License</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../about.cpp" line="53"/>
        <location filename="../about.cpp" line="62"/>
        <source>Changelog</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../about.cpp" line="54"/>
        <source>Cancel</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../about.cpp" line="74"/>
        <source>&amp;Close</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../main.cpp" line="66"/>
        <source>Error</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../main.cpp" line="67"/>
        <source>You seem to be logged in as root, please log out and log in as normal user to use this program.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../main.cpp" line="77"/>
        <source>Unable to get exclusive lock</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../main.cpp" line="78"/>
        <source>Another package management application (like Synaptic or apt-get), is already running. Please close that application first</source>
        <translation type="unfinished"></translation>
    </message>
</context>
</TS>
